src/js/script.js

```diff
@@ -45,21 +45,88 @@
       defaultValue: 1,
       defaultMin: 1,
       defaultMax: 9,
-    }
+    },
   };
 
   const templates = {
-    menuProduct: Handlebars.compile(document.querySelector(select.templateOf.menuProduct).innerHTML),
+    menuProduct: Handlebars.compile(
+      document.querySelector(select.templateOf.menuProduct).innerHTML
+    ),
   };
 
+  class Product {
+    constructor(id, data) {
+      const thisProduct = this;
+
+      thisProduct.id = id;
+      thisProduct.data = data;
+
+      thisProduct.renderInMenu();
+      thisProduct.initAccordion();
+
+      console.log('new product: ', thisProduct);
+    }
+    renderInMenu() {
+      const thisProduct = this;
+
+      const generetedHTML = templates.menuProduct(thisProduct.data);
+
+      thisProduct.element = utils.createDOMFromHTML(generetedHTML);
+
+      const menuContainer = document.querySelector(select.containerOf.menu);
+
+      menuContainer.appendChild(thisProduct.element);
+    }
+    initAccordion(){
+      const thisProduct = this;
+
+      const clickableTrigger = thisProduct.element.querySelector(select.menuProduct.clickable);
+
+      clickableTrigger.addEventListener('click', function(event) {
+      
+        event.preventDefault();
+
+        const activeProduct = document.querySelector(select.all.menuProductsActive);
+        console.log('activeProduct', activeProduct);
+        console.log(thisProduct.element);
+
+        if(activeProduct && activeProduct != thisProduct.element){
+
+          activeProduct.classList.remove('active');
+          console.log('if');
+
+        }
+        thisProduct.element.classList.toggle('active');
+        
+      });
+    }
+  }
+
   const app = {
-    init: function(){
+    initMenu: function () {
+      const thisApp = this;
+      console.log('thisApp.data: ', thisApp.data);
+
+      for (let productData in thisApp.data.products) {
+        new Product(productData, thisApp.data.products[productData]);
+      }
```

Comment:
Zamiast for ja używam konstrukcji foreach 
`[a, b].forEach(element => {
 zrób coś z elementem 
})
`